english/mystory.md

```diff
@@ -0,0 +1 @@
+We are all travelers in the wilderness of this world, and the best we can find in our travels is an honest friend.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.